pkgs/development/libraries/rutabaga_gfx/default.nix

```
@@ -13,9 +19,17 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-0RJDKzeU7U6hc6CLKks8QcRs3fxN+/LYUbB0t6W790M=";
   };
 
+  patches = [
+    # Make gfxstream optional
+    # https://chromium-review.googlesource.com/c/crosvm/crosvm/+/4860836
+    (fetchpatch {
+      url = "https://github.com/google/crosvm/commit/c3ad0e43eb12cbf737a6049e0134d483e337363f.patch";
```

Let's fetch this from upstream, not a mirror:

```suggestion
      url = "https://chromium.googlesource.com/crosvm/crosvm/+/c3ad0e43eb12cbf737a6049e0134d483e337363f%5E%21/?format=TEXT";
      decode = "base64 -d";
```